lib/ansible/modules/cloud/misc/virt_net.py

```diff
@@ -255,11 +258,13 @@ def modify(self, entryid, xml):
                     return True
             else:
                 # change the host
-                if host.get('name') == new_data.get('name') and host.get('ip') == new_data.get('ip'):
+                if host.get('name') == new_data.get('name')
+                and host.get('ip') == new_data.get('ip'):
```

Comment:
In this case, and similar ones, the `if` conditions need to be put in parentheses, for example:
```py
if (host.get('name') == new_data.get('name')
    and host.get('ip') == new_data.get('ip')):
```